app/api/helpers/events.py

```diff
@@ -28,3 +28,11 @@ def create_custom_forms_for_attendees(event):
     save_to_db(first_name_form, 'First name form saved')
     save_to_db(last_name_form, 'Last name form saved')
     save_to_db(email_form, 'Email form saved')
+
+
+def organizer_status(user, order):
```

Comment:
Move this method to event model and rename to `is_organizer` and it'll take user ID as parameter. 

If there already is a method to check permissions, use it

Reply:
@iamareebjamal Used aa predefined `is_organizer`